mon-pix/app/components/tutorial-item.js

```diff
@@ -3,12 +3,18 @@ import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
-const statusTypes = {
+const savingStatusTypes = {
   unsaved: 'unsaved',
   saving: 'saving',
   saved: 'saved',
 };
 
+const evaluationStatusTypes = {
+  unevaluated: 'unevaluated',
+  evaluating: 'evaluating',
+  evaluated: 'evaluated',
+};
```

Comment:
Effectivement, c'est un renommage que j'avais également prévu. Merci ! 
J'avais plutôt en tête une liste de statuts qui conviendrait aux deux boutons : 
```suggestion
const buttonStatusTypes = {
  unrecorded: 'unrecorded',
  pending: 'pending',
  recorded: 'recorded',
};
```

Reply:
Ah oui c'est plus simple, je change.